src/lib/VirtualScroller.js

```diff
@@ -421,6 +421,10 @@ class VirtualScroller {
             // will get rendered.
             const topPosition = (this.itemHeight + this.margin) * rowIndex;
             this.scrollingEl.scrollTop = topPosition;
+            // Some browsers don't fire the scroll event when setting scrollTop
+            // (IE11 & Firefox) so we need to manually dispatch the event
+            // in order to trigger `onScrollHandler` to render the items
+            this.scrollingEl.dispatchEvent(new Event('scroll'));
```

Comment:
Any issues with the event firing twice on browsers that automatically trigger it?

Reply:
I didn't see it firing twice, but I'll double check

Reply:
Yeah you're right it fires twice, but the `scrollTop` is the same so the logic to render items won't get triggered the second invocation